src/main/webapp/js/common.js

```diff
@@ -558,6 +558,30 @@ function scrollToTop(duration) {
 var DIV_STATUS_MESSAGE = '#statusMessagesToUser';
 
 /**
+ * Prepares a status message and the message status.
+ * Default message type is info.
+ *
+ * @param message the text message to be shown to the user
+ * @param status type
+ * @return created status message div
+ */
+function createStatusMessageDiv(message, status) {
```

Comment:
actually since we are not creating a div, shouldn't use `create`, I can't think of other better names, perhaps `populateMessageDiv`, `fillMessageDiv` or `writeToStatusMessageDiv`?